### PR TITLE
Incorporated features

### PR DESCRIPTION
**Pull request by artlov59**

### CODE REVIEW

File: documentation/closeout.md [hunk 1]
```diff
@@ -33,4 +36,22 @@ A few other organizational choices we made that were underutilized or hampered o
...
+### 4.1. Successfully Incorporated Features
+
+- Image wall (Users are able to see uploaded pictures on the main page).
+- Uploading images (Users are able to upload images on the main page).
+- Registration (Invited users are able to create a profile).
+- Login (Users that have been invited by other privileged users and have registered are able to log in).
+- Hiding messages in the pictures (Privileged users are able to embed hidden messages).
+- Creating invites (Priveleged users are able to invite other users).
+- Changing password (Privileged users are able to change their password).
+- Logout (Priveleged users are able to logout of their profile).
```
Comment (amitojsingh366) [inline documentation/closeout.md:52]: this is currently brokewn afaik

Comment (artlov59) [reply documentation/closeout.md:52]: Should i delete it then?

**Review by payrollinsights — APPROVED** [review]

File: apps/web/app/components/ThemeSwitcher.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const ThemeSwitcher = () => {
         setIsDark((dark) => {
             const newTheme = !dark;
             localStorage.setItem('@app/theme', newTheme ? 'dark' : 'light')
-            return !newTheme
+            return newTheme;
```
Comment (artlov59) [inline apps/web/app/components/ThemeSwitcher.tsx:21]:
```suggestion
            return !newTheme;
```

File: apps/web/app/components/ThemeSwitcher.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ export const ThemeSwitcher = () => {
         setIsDark((dark) => {
             const newTheme = !dark;
             localStorage.setItem('@app/theme', newTheme ? 'dark' : 'light')
-            return !newTheme
+            return !newTheme;
```
Comment (artlov59) [inline apps/web/app/components/ThemeSwitcher.tsx:21]:
```suggestion
            return !newTheme
```

**Pull request merged**